content/content-strategy/audit-content/plan-your-audit/toc.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: nav/sections
+sections:
+  - Talk about team benefits
+  - Align your goals with agency priorities
```

Comment:
This does not match the section name in `content/content-strategy/audit-content/prove-value/heading2.md`

Reply:
Trisha fixed